Packs/PaloAltoNetworks_IoT3rdParty/Playbooks/Incremental_Export_to_Cisco_ISE_-_PANW_IoT_3rd_Party_Integration_README.md

```diff
@@ -0,0 +1,31 @@
+Playbook to be run every 15 minutes via a job. Each run will get incremental updates for devices from PANW IoT cloud, and will update or create new endpoints in Cisco ISE with PANW IOT discovered attributes (ISE custom attributes)
```

Comment:
This playbook runs every 15 minutes as a scheduled job. Each run gets incremental device updates from PANW IoT Cloud and either updates existing endpoints or creates new endpoints in Cisco ISE.  New endpoints are created with PANW IoT discovered attributes (ISE custom attributes).

{Please review if the second sentence is technically accurate, that was my understanding. Are existing endpoints also updated with PANW IoT discovered attributes?}